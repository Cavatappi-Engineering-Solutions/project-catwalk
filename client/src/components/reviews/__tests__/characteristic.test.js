/* eslint-disable no-undef */
import React from 'react'
import renderer from 'react-test-renderer'
import Characteristic from '../characteristic.jsx'

it('renders correctly', () => {
  const tree = renderer.create(<Characteristic
    name={'test'}
    characteristics={{ value: 0, low: 'low', high: 'high' }}
  />).toJSON()
  expect(tree).toMatchSnapshot()
})
