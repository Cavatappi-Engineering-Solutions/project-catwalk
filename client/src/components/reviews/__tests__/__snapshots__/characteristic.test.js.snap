// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="characteristic"
>
  <span
    className="char-name"
  >
    test
  </span>
  <div>
    <div
      className="char-bar"
    />
    <div
      className="char-marker"
      style={
        Object {
          "--marker-position": "-3%",
        }
      }
    >
      â–¼
    </div>
  </div>
  <div
    className="char-meaning"
  >
    <span
      className="char-meaning-low"
    >
      low
    </span>
    <span
      className="char-meaning-high"
    >
      high
    </span>
  </div>
</div>
`;
